### ISSUE TITLE
[Feature] [App] Group by multiple params in charts

### ISSUE THREAD

**Issue by sophiaas** [wandb/client#2680]
**Is your feature request related to a problem? Please describe.**
When generating sweeps over multiple hyperparameters, I often want to group by multiple params in the charts in the app. For example, if I'm sweeping through learning rate, optimizer, and random seed, I often want to collapse across seed for error bars, but keep different learning_rate + optimizer combos as distinct lines.

**Describe the solution you'd like**
I'd love the option to group by multiple features! 

**Additional context**
If any of the above is unclear, please let me know!

**Comment by sophiaas** [comment]
Thanks for the suggestion @aidanjdonohue. I haven't yet, but will now. 

I do think there's a case to be made for adding this feature natively rather than having the user spin up a custom method -- I suspect that the majority of folks using wandb will be running many runs with different random seeds, and it's a minor edit to the existing infrastructure.

**Comment by aidanjdonohue** [comment]
That seems reasonable. We'll take a look at this more on our end.

**Comment by sophiaas** [comment]
Thank you!

**Comment by pkuderov** [comment]
+1 to the feature

Even more, I think a large portion of use cases could also be simplified with "!<param>" grouping feature allowing you to group by all params except the one parameter `<param>`. 

The obvious common use case is to collapse across random seeds. And, as the list of the parameter can be pretty long, it'd be really useful to specify only the exceptional single parameter instead of specifying all the rest.

**Comment by armanhar** [comment]
Hey Petr, thanks for sharing your feedback. I've bumped up the priority of the ticket. Will keep you notified of any updates.

**Comment by icedpanda** [comment]
any updates?
